Mike Hulbert/wk2/atm/js/atm.js

```diff
@@ -0,0 +1,121 @@
+
+$(document).ready(function(){
+  loadData();
+});
+
+const loadData = function (){
+  $queryAcc($checkingAcc, '#checking-balance'); // query checking acc
+  $queryAcc($savingsAcc, '#savings-balance'); // query savings acc
+  checkPattern('#checking-amount', '#checking-deposit', '#checking-withdraw');
+  checkPattern('#savings-amount', '#savings-deposit', '#savings-withdraw');
+}
+
+//************************* Deposit: Checking Account ************************//
+//See README.md before editing function
+
+let $checkingAcc = 0; // total balance in the checking account
+let $checkingDisplay = 0; // display current balance
+
+const $depositChecking = function (a, b) {
```

Comment:
After reading a bit more I realise that a is amount and b is balance, however this would be a little bit more readable if you had them as amount and balance. Readability is more important that brevity.